bundle/manifests/clusterlogging.clusterserviceversion.yaml

```diff
@@ -330,8 +330,70 @@ spec:
         x-descriptors:
         - urn:alm:descriptor:com.tectonic.ui:podStatuses
       version: v1
-    - kind: LogFileMetricExporter
+    - description: A Log File Metric Exporter instance. LogFileMetricExporter is the
+        Schema for the logFileMetricExporters API
+      displayName: Log File Metric Exporter
+      kind: LogFileMetricExporter
       name: logfilemetricexporters.logging.openshift.io
+      resources:
```

Comment:
I'm not too clear on the purpose of "resources" here, but if we go by https://github.com/operator-framework/operator-lifecycle-manager/blob/master/doc/design/building-your-csv.md it sounds like these are the "building blocks" of the CRD. In which case we don't need CronJob or PVC but need DaemonSet?

Reply:
Based on that design doc, I am inclined to include only `Pod`, `DaemonSet`, `Service`, and `ServiceMonitor` since those are the resources the LFME reconciles. Is this accurate?

Reply:
Yep, I'd say so.